website/get-involved/index.html

```diff
@@ -0,0 +1,172 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "_includes/base-resp.html" %}
+
+{% block page_title %}{{ _('Get Involved') }}{% endblock %}
+
+{% block page_desc %}
+{% if LANG.startswith('en-') %}
```

Comment:
Not sure why this is here, the whole page will be en-US only.

Reply:
It was left over from my paste of the features page. I likely left it in because I remembered your comment saying I should stick the gettext calls in, but forgot you had said to leave off that bit.